.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-  "semi": false,
+  "semi": true,
```

Comment:
Tried to check for prettier and every file had issues. Can yourun prettier --write . so everything is correct from here and husky can take over?